Author: Philip Rinn <rinni@inventati.org>
Description: Fix build with C++11 as char is unsigned on some architectures
Last-update: 2015-11-04
--- a/rtengine/dcraw.cc
+++ b/rtengine/dcraw.cc
@@ -2054,7 +2054,7 @@
 
 void CLASS kodak_radc_load_raw()
 {
-  static const char src[] = {
+  static const signed char src[] = {
     1,1, 2,3, 3,4, 4,2, 5,7, 6,5, 7,6, 7,8,
     1,0, 2,1, 3,3, 4,4, 5,2, 6,7, 7,6, 8,5, 8,8,
     2,1, 2,3, 3,0, 3,2, 3,4, 4,6, 5,5, 6,7, 6,8,
